### PR TITLE
Hud renovation

### PR DESCRIPTION
**Pull request by ghost**
Hud designed has been implemented in Unity

View of the HUD:
hudbuild

### CODE REVIEW

**Comment by quimix93** [comment]
The time is written in 2 lines, can you check it and show it in just one line?

**Comment by ghost** [comment]
Old screenshot, wait that I upload a new one

File: Assets/Scripts/Model/APIHUD.cs [hunk 1]
```diff
@@ -6,11 +6,27 @@ public class APIHUD : MonoBehaviour
...
+	public GameObject attackIndicator;
+	public GameObject defenseIndicator;
+	public GameObject speedIndicator;
+	public GameObject rangeIndicator;
+	public GameObject actionUpLeft;
+	public GameObject actionUpCenter;
+	public GameObject actionUpRight;
+	public GameObject actionDownLeft;
+	public GameObject actionDownCenter;
+	public GameObject actionDownRight;
```
Comment (Solero93) [inline Assets/Scripts/Model/APIHUD.cs:23]: Holy cow! Are all these really necessary?

File: Assets/Scripts/Model/APIHUD.cs [hunk 2]
```diff
@@ -19,137 +35,120 @@ private void Update()
...
     {
-        if (selectedItem == false)
-            setSelectedItemLabel();
-
-        var maxWidthBarLife =
-            transform.FindChild("containerStats")
-                .FindChild("container_info")
-                .FindChild("imgLifeBar")
-                .GetComponent<RectTransform>()
-                .rect.width;
```
Comment (Solero93) [inline Assets/Scripts/Model/APIHUD.cs:32]: Thank you for fixing all these @majosa. The dot-ception was an eyesore!

**Review by Solero93 — COMMENTED** [review]
I commented some stuff, but I don't wanna approve since I have no idea of this part of the code...

**Comment by Solero93** [comment]
> Old screenshot, wait that I upload a new one

The screenshot that's in this PR is the new one or still the old one?

**Comment by ghost** [comment]
That's the new one

**Comment by ghost** [comment]
Branch has been rejected due that contains unauthorized code

**Pull request closed without merging**